README.md

```diff
@@ -191,6 +191,23 @@ To access the admin web UI of Rspamd point the browser to
 
     https://<mail host>/rspamd/
 
+## Rspamd bulk Bayesian filter training
+
+To quickly train the Rspamd Bayesian filter plugin, execute the `rspamc`
+wrapper command, which incorporates the Rspamd authentication header
+logic. For example, get `rspamc` help output with:
+
+    runagent -m mail1 podman exec -i dovecot rspamc-wrapper --help
+
```

Comment:
`runagent -m mail1 podman exec -i dovecot  /usr/local/lib/dovecot/sieve-pipe/rspamc-wrapper --help`

the path is not found

```
[mail1@R1 toto]$ runagent -m mail1 podman exec -i dovecot rspamc-wrapper --help
Error: crun: executable file `rspamc-wrapper` not found in $PATH: No such file or directory: OCI runtime attempted to invoke a command that was not found
```